ci/build-and-send-slack-notification.ps1

```diff
@@ -0,0 +1,77 @@
+# Send a Slack notification with a link to the build.
+
+# Download previously uploaded slack attachments that were generated for each testing step
+New-Item -ItemType Directory -Path "$PSScriptRoot/slack_attachments"
+& buildkite-agent artifact download "*slack_attachment_*.json" "$PSScriptRoot/slack_attachments"
+
+$attachments = @()
+$all_steps_passed = $true
+Get-ChildItem -Recurse "$PSScriptRoot/slack_attachments" -Filter *.json | Foreach-Object {
+    $attachment = Get-Content -Path $_.FullName | Out-String | ConvertFrom-Json
+    if ($attachment.color -eq "danger") {
```

Comment:
Is depending on the `color` property too brittle?

Reply:
I think it's fine. You could also search for "error"